layouts/partials/head.html

```diff
@@ -11,6 +11,19 @@
 </script>
 <meta charset="utf-8">
 <title>{{ if .Title }}{{ .Title }} - {{ end }}{{ site.Title }}</title>
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "Organization",
+    "url": "https://kubernetes.io",
+    "logo": "https://kubernetes.io/images/favicon.png",
+    "potentialAction": {
+      "@type": "SearchAction",
+      "target": "https://kubernetes.io/docs/search/?q={search_term_string}",
```

Comment:
Should this be included for all revisions of the docs, or just the current release?
(when we cut the next release, will this cause problems?)

Reply:
I think in all releases. I'm targetting master for it.

Reply:
I'd expect to list a SearchAction just for the current release.
(I'm thinking of: use a shortcode to add that `application/ld+json` script block, and have the shortcode render to empty-string unless it's building the _master_ branch).

Reply:
Is it supported by the search page? The JSON+LD is just a shortcut to this page. If the search page does not support it neither will the JSON+LD. I'm asking because if I go on any other version other than current the search is not working. I propose we merge that so that at least the current version is supported. If you want to add support to the previous release I would suggest doing that in a different PR.

Reply:
The problem I envisage is that the metadata for one docs release suggests search the docs for a different release: a poor user experience. I'd like to do it right the first time.

I think I could help with the code for that too.

Reply:
Could you show me an example of it happening? When I type: https://kubernetes.io/docs/search/?q=deployment I don't see the documentation for the previous releases.

Reply:
The problem I'm picturing happens when v1.18 is out and someone goes to https://v1-17.docs.kubernetes.io/ then tries to search that.

Reply:
Why would it happen? Did it happen for v1.17 is out and someone went to https://v1-16.docs.kubernetes.io/. BTW I don't find any search features for old release.

Reply:
There _is_ a search page at https://v1-16.docs.kubernetes.io/docs/home/

The search page there doesn't exactly work; even so, it's wrong to suggest that `https://kubernetes.io/docs/search/?q={search_term_string}` is the search template for https://v1-16.docs.kubernetes.io/

Reply:
I think I could make a shortcode that excludes the content if building an old release.

Reply:
Could this happen in a different PR? And could we get this one merged?